.github/workflows/publish_breadbox_client.yaml

```diff
@@ -3,52 +3,24 @@ name: Publish breadbox client
 
 on:
   workflow_dispatch:
-    inputs:
-      tag:
-        description: "The tag/branch to checkout"
-        required: true
-        default: "master"
+
 jobs:
   publish-breadbox-client:
     runs-on: ubuntu-latest
     steps:
-      - name: Check out
+      - name: "Check out"
         uses: actions/checkout@v3
         with:
           token: "${{ secrets.GITHUB_TOKEN }}"
           fetch-depth: 0
-      - run: |
-          git pull origin HEAD:${{ inputs.tag }}
-      - name: Authenticate to Google Cloud
-        uses: google-github-actions/auth@v2
-        with:
-          # See instructions here: https://github.com/google-github-actions/auth?tab=readme-ov-file#service-account-key-json
-          credentials_json: ${{ secrets.DEPMAP_ARTIFACTS_SVC_ACCT }}
-      - name: Install and configure Poetry
-        uses: snok/install-poetry@v1
-        with:
-          version: 1.7.1
-          virtualenvs-create: true
-          virtualenvs-in-project: true
-      - name: Set up cache
-        uses: actions/cache@v2
-        id: cached-poetry-dependencies
-        with:
-          path: .venv
-          key: venv-${{ runner.os }}-${{ steps.full-python-version.outputs.version }}-${{ hashFiles('**/poetry.lock') }}
-      - name: Generate breadbox client
-        uses: ./.github/actions/generate-breadbox-client # defined as a re-usable action
-        # the above does a checkout which deletes the credential file, so we need to execute authenticate to google cloud after it
-      - name: Authenticate to Google Cloud
+
+      - name: "Authenticate to Google Cloud"
         uses: google-github-actions/auth@v2
```

Comment:
I'm guessing the warning below would go away if you updated this to `google-github-actions/auth@v2.1.7` (and did the same in `build_breadbox.yml`). 

And I'm curious if we added CodeQL to our repo, or is this something github added on their own? I've never seen a warning like that before

Reply:
I didn't add it. My guess was maybe broad IT sec might have added it.

Reply:
That's funny that adding a minor version would silence this warning... Adding a commit SHA is a cryptographic signature of the code ... but adding a minor version is a mutable tag like any other tag. I'm not sure how a minor version would prevent a supply chain attack. (That being said, I'm not especially worried about a supply chain attack from google, so I'm happy to do it just to silence the error)